liftwing/models/revertrisk.py

```diff
@@ -1,28 +1,29 @@
-import requests
+from pydantic import BaseModel, Field, model_validator
 from .liftwing_model import LiftwingModel
+from ..utils.validator import validate_payload, check_required_fields
 from typing import Any, Dict
 
 
-class RevertRiskAPIModel(LiftwingModel):
-    def __init__(self, base_url="https://api.wikimedia.org/service/lw/inference/v1/models/revertrisk-language-agnostic:predict"):
-        super().__init__(base_url)
+class RevertRiskPayload(BaseModel):
+    lang: str = Field(...,
+                      title="Language code of the wiki",
+                      description='A string representing the language code related to the target '
+                                  'wiki. Example: "en" for English Wikipedia.')
+    rev_id: int = Field(...,
+                        title="Revision ID of the edit",
+                        description="The revision id for the wiki identified by the lang parameter.")
 
-    def request(self, payload: Dict[str, Any], method: str = "POST", headers: Dict[str, str] = None) -> Dict[str, Any]:
-        """
-        This function makes a POST request to https://api.wikimedia.org/service/lw/inference/v1/models/revertrisk-language-agnostic:predict
-        using the language parameter and returns a JSON
-        language is for the different wiki languages, rev_id is the specific revisions
-        """
-        language = payload.get("lang")
-        if language is None:
-            raise ValueError("'lang' parameter is required in the payload.")
-        rev_id = payload.get("rev_id")
-        if rev_id is None:
-            raise ValueError("rev_id is none, add revision id to continue")
+    @model_validator(mode='before')
+    def check_fields(cls, values):
+        check_required_fields(values, ['lang', 'rev_id'])
+        return values
```

Comment:
We don't need to check required fields manually, since a field in pydantic without a default value is considered required.

Reply:
You're right. I never actually noticed the "lang" in the ValidationError, so I added the rest in order to have a mroeo informative error message.
I'd just remove it then and we can figure out a way for more informative error messages in the future.